docs/Options.md

```diff
@@ -40,6 +40,13 @@ These options are required to run osde2e.
 
 - Type: `string`
 
+### `POLLING_TIMEOUT`
+
+- PollingTimeout is how long to wait for an object to be created before
```

Comment:
Small discussion point: Is this for "objects" or for-specifically operators? Should we rename this completely to be operator-specific on the likelihood we will have additional objects we'll want to create-or-timeout with?

Reply:
Right now it's only being used for operators. I'm happy to rename if there is a better suggestion.

Reply:
It might warrant renaming later, for now I think it's reasonable. It was mainly a point for discussion. Thoughts @meowfaceman?